framework/demuxer/demuxer_service.cpp

```diff
@@ -287,6 +287,15 @@ namespace Cicada {
         return mDemuxerPtr->GetRemainSegmentCount(index);
     }
 
+    bool demuxer_service::isRealTimeStream(int index)
+    {
+        if (nullptr == mDemuxerPtr) {
+            return false;
+        }
+
+        return (!mDemuxerPtr->isPlayList()) || mDemuxerPtr->isRealTimeStream(index);
```

Comment:
1. If not a playList demuxer, then it is a realtime stream ?
2. Is the parameter index usefull?